package.json

```diff
@@ -101,10 +101,10 @@
     "less-loader": "^11.1.0",
     "log-symbols": "^4.1.0",
     "ora": "^5.4.1",
-    "postcss": "^8.4.23",
-    "postcss-discard-comments": "^6.0.0",
-    "postcss-loader": "^7.2.4",
-    "postcss-preset-env": "^8.3.2",
+    "postcss": "^8.4.39",
```

Comment:
can we add information about minimum supported nodejs version in this package.json?

Reply:
Done